datajoint/condition.py

```diff
@@ -61,6 +63,17 @@ def append(self, restriction):
             super().append(restriction)
 
 
+@dataclass
+class Top:
+    """
+    doc string
```

Comment:
@dimitri-yatsenko Any suggestions for the doc string?

Reply:
You can refer to the documentation page and the original issue / PR that documents the feature.

Reply:
https://github.com/datajoint/datajoint-python/pull/1084/files#diff-27f86ee6bd86a0f1f7ffaa9ae916330d6ec989e73ae0db9215801892ac30989cR68-R71